anybox/recipe/odoo/server.py

```diff
@@ -54,6 +54,27 @@ def __init__(self, *a, **kw):
             sw_modules = ('web', ) + sw_modules
         self.server_wide_modules = sw_modules
 
+        def replace_python_header(new_header):
+            """ Update the script template with the new header. """
+            zc.buildout.easy_install.script_template = (
+                zc.buildout.easy_install.script_template.replace(
+                    zc.buildout.easy_install.script_header, new_header))
+            zc.buildout.easy_install.py_script_template = (
+                zc.buildout.easy_install.py_script_template.replace(
+                    zc.buildout.easy_install.script_header, new_header))
+
+        if self.python_scripts_executable:
+            # Monkeypatch the script headers to replace the python
+            # executable by the one configured by the user
+            new_header = '#!%s' % self.python_scripts_executable
+            replace_python_header(new_header)
+
+        elif self._relative_paths and self.buildout_dir in sys.executable:
+            # Virtualenv usage in a folder inside the project folder, use
+            # relative path to the python executable
+            new_header = os.path.relpath(sys.executable)
+            replace_python_header(new_header)
```

Comment:
I am missing the `#!` she-bang when I build it like this, so please add it to the header.

Can you add a note about this behavior in the documentation under `relative-paths` below, and mention the gotcha that in this case, you need to call the bin/ scripts from the buildout directory (because I don't think we want to take any suggested approaches from here: https://stackoverflow.com/questions/20095351/shebang-use-interpreter-relative-to-the-script-path)

Reply:
@StefanRijnhart I do find that my addition is rather counter productive. It means that using absolute paths to the script also break, which is a more much more common use case. Perhaps we should revert this change?